rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -252,10 +255,15 @@ public int renewTimeLimit(URL lraId, Long timelimit) {
         return lra.setTimeLimit(timelimit);
     }
 
+    public boolean isLocal(URL lraId) {
+        return hasTransaction(lraId.toExternalForm());
+    }
+
     // TODO why doesn't this get triggered after construction
 //    @PostConstruct
     // TODO enableRecovery(@Observes @Initialized(ApplicationScoped.class) Object init)
-    public void enableRecovery() {
+    public void enableRecovery(@Observes @Initialized(ApplicationScoped.class)
+                                       Object init /* a javax.servlet.ServletContext */) {
```

Comment:
It's just below but why do we need to call RecoveryManager.manager().scan()

Reply:
I was going to implement an on demand scan (via calling one of the jax-rs resource methods). So this can be removed. I will include it in the "savePendingList" change that I will commit this morning.

Reply:
And the TODO just below is to re-populate the known list of "recovery coordinators". This is only required if a participant wants to tell us that it has moved (or to trigger a "replay" call). I left it as a TODO for now since we aren't using in any tests yet. I might add it this afternoon after taking a look at https://issues.jboss.org/browse/JBEAP-13023